vimeo-networking/src/main/java/com/vimeo/networking/model/InteractionCollection.java

```diff
@@ -134,6 +143,10 @@ public void setBuy(@Nullable Interaction buy) {
         mBuy = buy;
     }
 
+    public void setPurchase(@Nullable Interaction purchase) {
```

Comment:
Why is this public?

Reply:
Because the following setters in this model are also `public`:
```java
    public void setLike(@Nullable Interaction like)
    public void setWatchLater(@Nullable Interaction watchLater)
    public void setFollow(@Nullable Interaction follow)
    public void setBuy(@Nullable Interaction buy)
    public void setRent(@Nullable Interaction rent)
    public void setSubscribe(@Nullable Interaction subscribe)
```